### ISSUE TITLE
Chemistry book disappears from zoomed-in view

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/friction#267]
**Test device**
MacBook Air (m1 chip)

**Operating System**
11.6

**Browser**
chrome and safari

**Problem description**
https://github.com/phetsims/qa/issues/717

This issue is only seen on Master and not in this dev version.
Using keyboard nav, it is possible for the entire chemistry book to move out of zoomed-in view. 

**Steps to reproduce**
1. Use keyboard nav to select the zoomed-in chemistry book
2. Hold down either the left or right arrow key.

**Visuals**
![bookleaves](https://user-images.githubusercontent.com/87318828/138717887-77645f9c-15d1-4940-ba15-e52476658499.gif)

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: Friction
URL: https://bayes.colorado.edu/dev/phettest/friction/friction_en.html?ea&brand=phet
Version: 1.6.0-dev.19 (unbuilt)
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.54 Safari/537.36
Language: en-US
Window: 1431x690
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by zepumph** [comment]
>Yes, this is a known issue that I haven't gotten to from work in #239.

I'm actually just going to close this as a duplicate, I  think it will make things simpler. Thanks!
